### ISSUE TITLE
Keybase App on macOS Monterey Beta - Finder Integration Broken

### ISSUE THREAD

**Issue by martynbristow** [keybase/client#24578]
Finder integration is broken on macOS Monteray

When Renenabling finder integration, I get the pop up which spins and then comes back to the initial dialog
<img width="824" alt="image" src="https://user-images.githubusercontent.com/3476872/126896667-9b14b233-3ee2-4cea-ae32-a21117dc8c06.png">

But then I have KeyBase in the sidebar (again):

**I had removed it before reenabling**

I get this error

<img width="584" alt="image" src="https://user-images.githubusercontent.com/3476872/126896703-0a877e28-909e-4e07-9ae3-ce32a0303f81.png">

my log id: 7313ae3d3b931cac9aeaff1c

**Comment by cyli** [comment]
I have the same issue in Mac OS Big Sur 11.5.1 after the last update.

**Comment by danielhavir** [comment]
Similar issue, running macFUSE 4.2.1 and macOS 12.0.1 and keybase version 5.8.0

**Comment by glyph** [comment]
Same issue, same versions as @danielhavir.  macFUSE 4.2.1 is loaded, I verified that sshfs works. kbfs doesn't.  In "Security & Privacy", I see this:
<img width="780" alt="image" src="https://user-images.githubusercontent.com/716529/139572870-88d6ea29-c6a3-4e5e-a6e9-d998e87e3c17.png">
Clicking "Open Anyway" does nothing.

Attempting to manually load the extension does this:

```
Executing: /usr/bin/kmutil load -p /Library/Filesystems/kbfuse.fs/Contents/Extensions/11/kbfuse.kext
Error Domain=KMErrorDomain Code=71 "Kernel request failed: (libkern/kext) kext (kmod) start/stop routine failed (-603946985)" UserInfo={NSLocalizedDescription=Kernel request failed: (libkern/kext) kext (kmod) start/stop routine failed (-603946985)}
```

Console logs show:

```
default	00:25:46.317487-0700	kernel	Failed to load kext com.github.kbfuse.filesystems.kbfuse (error 0xdc008017).
default	00:25:46.321892-0700	kernelmanagerd	(kernel): Kext com.github.kbfuse.filesystems.kbfuse did not start (return code 0x5).
error	00:25:46.321921-0700	kernelmanagerd	(kernel): Kext com.github.kbfuse.filesystems.kbfuse start failed (result 0x5).
error	00:25:46.321938-0700	kernelmanagerd	(kernel): Kext com.github.kbfuse.filesystems.kbfuse failed to load (0xdc008017).
error	00:25:46.321959-0700	kernelmanagerd	(kernel): Failed to load kext com.github.kbfuse.filesystems.kbfuse (error 0xdc008017).
default	00:25:46.321968-0700	kernelmanagerd	Received kext unload notification: com.github.kbfuse.filesystems.kbfuse
```

This looks like maybe there's just literally no functioning `kbfuse` kernel extension for this kernel version yet?

**Comment by techieshark** [comment]
Same here. MacOS Monterey, version 12.0.1. 

Here's a screen video showing that something appears to be in progress, but nothing happens (and this can be repeated any number of times to no effect):

https://user-images.githubusercontent.com/1072292/139752619-b8b93f73-ca8e-4214-867f-1858b21e8a7b.mp4

Not sure if it matters, but here are the contents of kbfuse.fs:

```shell
ls /Library/Filesystems/kbfuse.fs/Contents/Extensions/
10.14 10.15 10.16 11
```

When I click Keybase in the finder, an error message says "The folder (Keybase) cannot be opened be opened because you don't have permissions to see its contents."

**Comment by asantoz** [comment]
> Same here. MacOS Monterey, version 12.0.1.
> 
> Here's a screen video showing that something appears to be in progress, but nothing happens (and this can be repeated any number of times to no effect):
> 
>  keybase-fuse-install-does-nothing.mp4 
> Not sure if it matters, but here are the contents of kbfuse.fs:
> 
> ```shell
> ls /Library/Filesystems/kbfuse.fs/Contents/Extensions/
> 10.14 10.15 10.16 11
> ```
> 
> When I click Keybase in the finder, an error message says "The folder (Keybase) cannot be opened be opened because you don't have permissions to see its contents."

Having the same issues, there is any workaround to access the files instead of Finder?

**Comment by jeromelaurens** [comment]
same here

**Comment by maxtaco** [comment]
Closed as duplicative of #24636
